### ISSUE TITLE
[rush] rush update throws error - Invalid tag name

### ISSUE THREAD

**Issue by sadsa** [microsoft/rushstack#1547]
**Is this a feature or a bug?**

- [ ] Feature
- [x] Bug

**Please describe the actual behavior.**

When running `rush update` or `rush install` I get the follow error: 

```
Starting "rush install"


Found files in the "common/git-hooks" folder.
Successfully installed these Git hook scripts: pre-commit

Trying to acquire lock for pnpm-3.8.1
Acquired lock for pnpm-3.8.1
Found pnpm version 3.8.1 in C:\Users\--\.rush\node-v10.16.1\pnpm-3.8.1

Symlinking "C:\Users\--\Projects\--\arc-mono\common\temp\pnpm-local"
  --> "C:\Users\--\.rush\node-v10.16.1\pnpm-3.8.1"

Updating temp projects in C:\Users\--\Projects\--\arc-mono\common\temp\projects

ERROR: Invalid tag name "1.8.1_prettier@1.15.3": Tags may not have any characters that encodeURIComponent encodes.

```

**What is the expected behavior?**

I'm not sure where this error is coming from nor how to resolve it. Curiously, deleting the `pnpm-lock.yaml` and running `rush update` file seems to resolve the issue.

* **Tool: Rush 5.13.0** 
* **Tool Version: 5.13.0** 
* **Node Version: 10.16.1** 
  * **Is this a LTS version? Yes** 
  * **Have you tested on a LTS version? Yes** 
* **OS: Windows 10**

**Comment by iclanton** [comment]
@sachinjoseph is working on fixing this issue in https://github.com/microsoft/web-build-tools/pull/1542. Can you try this on Rush 5.11.X?

**Comment by KevinTCoughlin** [comment]
> @sachinjoseph is working on fixing this issue in #1542. Can you try this on Rush 5.11.X?

@iclanton, I have an internal repro of this exact bug with Rush `5.13.0`. It does not occur with Rush `5.11.0`. We can use that to validate a fix if need be.

**Comment by sadsa** [comment]
I can confirm that Rush 5.11.0 does not throw the error

**Comment by rakeshpatnaik** [comment]
@sadsa 
The issue is fixed in 5.13.1. Please check and let us know if you still see the issue.

**Comment by KevinTCoughlin** [comment]
> @sadsa
> The issue is fixed in 5.13.1. Please check and let us know if you still see the issue.

I can confirm upgrading to `5.13.1` fixed the issue for us. Thanks @sachinjoseph!
